src/app/map/map.component.spec.ts

```diff
@@ -350,17 +350,6 @@ describe('MapComponent', () => {
         component.updateEventFilter();
     });
 
-    it('#eventFocus sets map to event focused view', () => {
```

Comment:
Not permanently deleted - I added a version of this test back in the issue I'm currently working on